### ISSUE TITLE
Error in loading Chunks (Nether and End) - Dev Build

### ISSUE THREAD

**Issue by OldVersion24** [ViaVersion/ViaBackwards#630]
### '/viaversion dump' Output

https://dump.viaversion.com/b9282ff6bea36062b398b85dd1cd9e6888df60bc3f01bb01ce509394b4f36b16

### Console Error

There are no console errors.

### Bug Description

Well, I had kicks on 1.7.x with my server being 1.8.x.

Before opening a bug, I tried the build dev, and yes, it fixed the kicks, the problem was when I changed the world either to End or Nether, the chunks loaded very very rare (https://prnt.sc/uuiCWDX77nSW here an image). Then I guessed that it was that in some build dev, the worlds mentioned above are broken. And yes, indeed, in build 3.0.2-Snapshot+29 the problems with the "Rewrote chunk handling (1.7--1.8)" started.

How do I know, well, I just put the 3.0.2-Snapshot+28, 1 build dev less, and with that yes, the worlds were fixed, but some kicks in 1.7 were still there, so I assume that in higher build devs the kicks were fixed, but they spoil the nether and end worlds.

Well, to reproduce it you only have to change the world and verify what I say, in the same way there I indicated that in that snapshot the chunks are touched, and in the same way I checked that in the snapshot number "3.0.3-SNAPSHOT+49" the chunks "Updated chunk reading and writing to work with Via again" are touched again.

That's all, thank you very much.

### Steps to Reproduce

Just use a ViaRewind dev build higher than +28 and switch to the worlds I mentioned (Nether or End).

### Expected Behavior

So that the chunks for those of 1.7 load well, since only the world of the overworld loads well.

### Additional Server Info

Yes, this does not happen to players of 1.8 or higher, they do see the chunks well.

### Checklist

- [X] Via plugins are only running on **EITHER** the backend servers (e.g. Paper) **OR** the proxy (e.g. BungeeCord), **not on both**.
- [X] I have included a ViaVersion dump.
- [X] If applicable, I have included a paste (**not a screenshot**) of the error.
- [X] I have tried a build from https://ci.viaversion.com/ and the issue still persists.

**Comment by Barvalg** [comment]
![Platform: AstralMC%201.8.8%20%28MC%3A%201.8.8%29](https://img.shields.io/badge/Platform-AstralMC%201.8.8%20%28MC%3A%201.8.8%29-3498db?style=flat-square) </br>![ViaVersion (4.8.1): 2 commits behind master](https://img.shields.io/badge/ViaVersion%20(4.8.1)-2%20commit(s)%20behind%20master-yellow?style=flat-square) </br>![ViaRewind(3.0.2-SNAPSHOT): 43 commits behind dev](https://img.shields.io/badge/ViaRewind%20(3.0.2--SNAPSHOT)-43%20commit(s)%20behind%20dev-yellow?style=flat-square) </br><br><h3>Please try the **latest** build(s) from [https://ci.viaversion.com/](https://ci.viaversion.com/). In case the issue still persists send the new dump</h3>

**Comment by OldVersion24** [comment]
Yes of course, we tested it just now, I'm sending the new one, dump, it's the same: https://dump.viaversion.com/c131a186eeaba007aec3fdc3b559a689276d6ced317563c481b4525b6a5a0011

**Comment by Barvalg** [comment]
![Platform: AstralMC%201.8.8%20%28MC%3A%201.8.8%29](https://img.shields.io/badge/Platform-AstralMC%201.8.8%20%28MC%3A%201.8.8%29-3498db?style=flat-square) </br>![ViaVersion (4.9.0-23w44a-SNAPSHOT): Even with dev](https://img.shields.io/badge/ViaVersion%20(4.9.0--23w44a--SNAPSHOT)-Even%20with%20dev-brightgreen?style=flat-square) </br>![ViaRewind(3.0.4-SNAPSHOT): Even with dev](https://img.shields.io/badge/ViaRewind%20(3.0.4--SNAPSHOT)-Even%20with%20dev-brightgreen?style=flat-square) </br>
